src/sparseml/pytorch/sparsification/quantization/helpers.py

```diff
@@ -588,44 +619,6 @@ def get_observer(
     return observer
 
 
-def fix_observer_quant_range(module: Module):
-    """
-    As of torch 1.10.2 there is a bug in FakeQuantize initialization where
-    quant_min and quant_max of FakeQuantize are not propagated to its
-    activation_post_process observer. This function propagates FakeQuantize quant
-    ranges to their Observer objects
```

Comment:
It's only safe to remove this function if we always use non-customized ranges. Otherwise I can do a functools.partial above to properly pass the values